addons/product/views/product_attribute_views.xml

```diff
@@ -91,7 +91,7 @@
         <field name="model">product.template.attribute.value</field>
         <field name="type">tree</field>
         <field name="arch" type="xml">
-            <tree string="Attributes" create="0" delete="0">
+            <tree string="Attributes" create="0" delete="0" editable="bottom">
```

Comment:
How do you open the form view in that case? To set exclusions rules etc.

Also make sure all fields are readonly there except price_extra.